help/de/Help_ch_Account-Actions.xml

```diff
@@ -2469,4 +2304,552 @@ and update the appendix. -->
+        Datenausgabe im <guilabel>XML</guilabel> oder <guilabel>SQLite3</guilabel>-Format anbietet. Haben
+        Sie bereits konfigurierte Datenbank-Systeme installiert, so werden diese ebenfalls hier zur
+        Auswahl angeboten.
+      </para>
+
+      <table id="Account-Export-Dialog">
+        <title>Dialog zum Exportieren von Konten</title>
+
+        <tgroup align="left" cols="2">
+          <tbody>
+            <row>
+              <entry>
+                <para><figure>
+                    <title>Exportieren in Datei</title><screenshot>
+                      <mediaobject>
+                        <imageobject>
+                          <imagedata fileref="figures/AccountTree_Export_file.png" 
+                         srccredit="Frank H. Ellenberger" width="100%"></imagedata>
+                        </imageobject>
+                      </mediaobject>
+                    </screenshot>
+                  </figure>
+                </para>
+              </entry>
+
+              <entry>
+                <para><figure>
+                    <title>Exportieren in Datenbank</title><screenshot>
+                      <mediaobject>
+                        <imageobject>
+                          <imagedata fileref="figures/AccountTree_Export_dbms.png" 
+                         srccredit="Frank H. Ellenberger" width="100%"></imagedata>
+                        </imageobject>
+                      </mediaobject>
+                    </screenshot>
+                  </figure>
+                </para>
+              </entry>
+            </row>
+          </tbody>
+        </tgroup>
+      </table>
+
+      <variablelist spacing="compact">
+        <varlistentry>
+          <term><guilabel>XML</guilabel></term>
+
+          <term><guilabel>SQLite3</guilabel></term>
+
+          <listitem>
+            <para>In dem Dateiauswahldialog können Sie Angaben zu dem Dateinamen und zu dem Verzeichnis/Ordner, in
+              dem die Datei gespeichert werden soll, machen. Mit Betätigen der Schaltfläche
+              <guilabel>Speichern unter…</guilabel> wird die Datei geschrieben. Sie haben hierbei
+              keinen weiteren Einfluss auf das Ergebnis der Exportdatei.
+            </para>
+          </listitem>
+        </varlistentry>
+
+        <varlistentry>
+          <term><guilabel>mysql</guilabel></term>
+
+          <term><guilabel>postgres</guilabel></term>
+
+          <listitem>
+            <para>Die Ansicht des Dialogs ändert sich und es werden Eingabefelder für die Informationen zur
+              Datenbankverbindung abgefragt. Wenn Sie Ihre &app;-Daten in einem Datenbanksystem
+              speichern möchten, so benötigen Sie schreibenden Zugriff auf eine vorhandene
+              Datenbank oder die Berechtigung eine neue Datenbank anzulegen.
+            </para>
+          </listitem>
+        </varlistentry>
+      </variablelist>
+
```

Comment:
Vielleicht mal für später vormerken:
Diese schöne Beschreibung als `Dialog zum Datei speichern oder exportieren`
in Tools verschieben und von den anderen Vorkomnissen wie z.B. Neue-Datei-Assistent letzter Schritt oder Sitzungsende drauf verweisen.